.golangci.yml

```diff
@@ -0,0 +1,121 @@
+issues:
+  # List of regexps of issue texts to exclude, empty list by default.
+  # But independently from this option we use default exclude patterns,
+  # it can be disabled by `exclude-use-default: false`. To list all
+  # excluded by default patterns execute `golangci-lint run --help`
+
+  exclude-rules:
+    - linters:
+        - goimports
+      text: "File is not `goimports`-ed"
```

Comment:
What is this doing ?

🤔 is this conflicting with line 29 which is calling a `    - goimports`  linter ?

Reply:
( I think enforcing goimports is really nice )

Reply:
It is ignoring all files that don't have their packages properly formatted 

```
import (
  log
  github.com/hashicorp/packer/...
  context
  ...
)
```

There were a lot of violations when I first ran, and I didn't know what it meant then so I disabled to start :smile: But yeah this should be removed once we merge this and the violations are fixed. Adding to the todo list.

Reply:
Gotcha 👍 awesome :D :D :D I've manually done this once in the past but it has gotten way worse again ! I think `goimports -w *.go */*.go` works ( on `zsh` )